src/ace/managers/advancedsprite.c

```diff
@@ -0,0 +1,259 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ #include <ace/managers/advancedsprite.h>
+ #include <ace/utils/bitmap.h>
+ #include <ace/managers/blit.h>
+ #include <ace/utils/sprite.h>
+ #include <ace/managers/system.h>
+ 
+ 
+ #define SPRITE_WIDTH 16
+ #define TWOBPP_BYTEWIDTH 2
+ 
+ tAdvancedSprite *advancedSpriteAdd(UBYTE ubChannelIndex, UWORD uwSpriteHeight,tBitMap *pSpriteVerticalStripBitmap1, tBitMap *pSpriteVerticalStripBitmap2 ) {
```

Comment:
Remove the leading space here and across the whole file.

VSCode started to do this ugly stuff because of license's block comment - because of that I think I'll change the license comment to `//` style across the whole project soon.